### PR TITLE
Add edge case tests for DeleteCommand and DeleteCommandParser

### LINKED ISSUE #115: Update DeleteCommandTest & respective Parser to ensure testing of edge cases
Delete Command seems to work fine, but the parser tests seem a little non-extensive. There should be more tests to cover the different edge cases of this command to ensure bug-free functionality in the long run.

### PR DESCRIPTION
**Pull request by bipplane**
Fixes #115

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/DeleteCommandParserTest.java [hunk 1]
```diff
@@ -29,4 +38,35 @@ public void parse_validArgs_returnsDeleteCommand() {
     public void parse_invalidArgs_throwsParseException() {
         assertParseFailure(parser, "a", String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE));
     }
+
+    @Test
+    public void parse_leadingAndTrailingWhitespace_returnsDeleteCommand() {
```
Comment (izniy) [inline src/test/java/seedu/address/logic/parser/DeleteCommandParserTest.java:43]: Good addition, the new tests covers the additional possible variations in paths

File: src/test/java/seedu/address/logic/parser/DeleteCommandParserTest.java [hunk 2]
```diff
@@ -29,4 +38,35 @@ public void parse_validArgs_returnsDeleteCommand() {
...
+
+        expectedModel.deletePerson(personToDelete);
+
+        String expectedMessage = String.format(DeleteCommand.MESSAGE_DELETE_PERSON_SUCCESS,
+            Messages.format(personToDelete));
+
+        assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
+
+        // Verify person size to ensure person was actually deleted
+        int expectedSize = model.getFilteredPersonList().size();
```
Comment (izniy) [inline src/test/java/seedu/address/logic/parser/DeleteCommandParserTest.java:68]: Good use of size to check if person was deleted successfully

**Review by izniy — APPROVED** [review]
LGTM, good coverage of edge cases!

**Review by flljy940 — APPROVED** [review]
LGTM
